docs/multiple dataplane_CN.md

```diff
@@ -0,0 +1,205 @@
+# 多网格数据面协同设计文档
+
+## 背景
+
+传统的serviceMesh以namespace为粒度来使能serviceMesh功能。在不同的namespace中，可能会使用不同的serviceMesh软件。Kmesh作为serviceMesh的数据面软件之一，在使用过程中，可能与不同的serviceMesh数据面协同工作，本文档主要对Kmesh作为数据面软件与其他serviceMesh数据面软件进行协同工作的场景进行描述。
+
+## 支持的使用场景
+
+Kmesh作为client端使用：
+
+1. 使用Kmesh纳管pod与普通pod进行通讯
+2. 双端都使用Kmesh进行通讯
+3. 使用Kmesh纳管与sidecar（envoy）网格数据面中的pod进行通讯
+
+Kmesh作为Server端使用：
+
+1. Kmesh仅在Server端有envoy注入标签时，通过iptables来短接envoy
+
+当前存在问题的场景：
+
+Kmesh当前不支持在Server端短接sidecar（envoy）并与使用sidecar（envoy）作为Client端的Pod进行协同通信，原因是因为Client上有sidecar（envoy）发送时，识别到Server端也有sidecar（envoy）时，会使用mTLS进行加密通讯，Kmesh当前无法进行解密，计划在后续版本中支持此特性。
+
+## 规格约束
+
+1. 使用Kmesh纳管Pod时基于namespace粒度
+2. Kmesh纳管namespace生效后，只影响新启动的Pod，对存量的Pod没有影响，Kmesh结束纳管同理。
+
+## 场景价值
+
+在同一个集群范围内，可能会部署多个网格数据面，Kmesh需要与无网格、其他类型网格代理的namespace中的Pod进行通讯，需要保证其中的Pod之间通信结果正常。
+
+## 用户场景一：Kmesh代理pod与普通pod进行通讯分析：
+
+场景描述
+
+- 纳管Pod上没有envoy
+	- 用户指定namespace使用Kmesh作为数据面
+		- 新启动Pod通过Kmesh编排寻址访问server端

```

Comment:
we may need to label or annotate the pod to indicate that kmesh is in charge of traffic governance. Otherwise, we could not know explicitly unless we investigate

Reply:
Agreed, this is necessary, I have already raised a [new requirement](https://github.com/kmesh-net/kmesh/issues/86) tracking ticket.